engine/main.py

```diff
@@ -0,0 +1,47 @@
+import math
+import threading
+import json
+
+from constants.definitions import ENGINE_PATH
+from engine.robot import Robot
+from engine.robot_state import Robot_State
+from engine.mission import Mission
+from engine.mission_state import Mission_State
+from engine.control_mode import ControlMode
+from engine.database import DataBase
+from engine.transmission import send_packet_to_gui
+from engine.is_raspberrypi import is_raspberrypi
+from engine.plot_trajectory import plot_sim_traj
+from engine.parser import parse_main
+
+if __name__ == "__main__":
+    # Load config from JSON file
+    with open(ENGINE_PATH +"/config.json", "r") as fp:
+        config_args = json.load(fp)
+    
+    # Load user-defined arguments
+    user_args = parse_main()
+
+    # Development config flags
+    is_transmit = config_args.get("is_transmit") # Set to true when the rpi/robot is communicating w/ the GUI
```

Comment:
remove is_transmit and just use config_args.get("is_transmit") wherever is_transmit is

Reply:
https://github.com/cornellnexus/src/pull/131#discussion_r1174086868